pkg/generate/app/app.go

```diff
@@ -258,6 +259,11 @@ func (r *ImageRef) ImageStream() (*imageapi.ImageStream, error) {
 	if !r.OutputImage {
 		stream.Spec.DockerImageRepository = r.String()
 		stream.Spec.Tags = map[string]imageapi.TagReference{}
+		if r.Insecure {
+			stream.ObjectMeta.Annotations = map[string]string{
+				"openshift.io/image.insecureRepository": "true",
```

Comment:
@mfojtik it is... I've updated this one to use the constant. Will push the update shortly